whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -156,6 +159,63 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
         return mapCommons(searchResult, requestUrl);
     }
 
+    public Object mapMergeSearch(final String requestUrl, List<RpslObject> objects,
+                                 final Iterable<LocalDateTime> localDateTimes, final int maxResultSize){
+        RdapObject organisation = new RdapObject();
```

Comment:
Use final keyword wherever possible

Reply:
we are setting the value of the organisation below, we can not use final

Reply:
OK. What happens if you don't find an organisation? Is an empty RdapObject() returned? Is that case handled elsewhere?